prebuilt-rules-scripts/generate.py

```diff
@@ -15,7 +15,7 @@
 PREBUILT_RULES = ROOT.joinpath('prebuilt-rules-scripts')
 GENERATED_ASCII = ROOT.joinpath('generated-ascii-files')
 DEFAULT_KIBANA_RULES_DIR = str(Path().joinpath('x-pack', 'plugins', 'security_solution', 'server', 'lib',
-                                               'detection_engine', 'rules', 'prepackaged_rules'))
+                                               'detection_engine', 'prebuilt_rules', 'content', 'prepackaged_rules'))
```

Comment:
Updated this so it points to the Kibana FS rules after recent change.

Reference -  https://github.com/elastic/kibana/pull/142950